app/views/schools/ects/show.html.erb

```diff
@@ -6,41 +6,66 @@
     caption_size: 'l',
   )
 %>
-<%= render Teachers::DetailsComponent.new(mode: :school, teacher: @ect.teacher) do |component|
-  component.with_personal_details
-  component.with_itt_details
-  component.with_induction_summary
-  component.with_current_induction_period
-  component.with_past_induction_periods
-end %>
-
-<h2 class='govuk-heading-m'>Additional details</h2>
+<h2 class='govuk-heading-m'>ECT details</h2>
 <%=
   govuk_summary_list(
     rows: [
-      { key: { text: 'Mentor' }, value: { text: ect_mentor_details(@ect) } },
+      { key: { text: 'Name' },              value: { text: teacher_full_name(@ect.teacher) } },
+      { key: { text: 'Email address' },     value: { text: @ect.email } },
+      { key: { text: 'Mentor' },            value: { text: ect_mentor_details(@ect) } },
       { key: { text: 'School start date' }, value: { text: ect_start_date(@ect) } },
-      { key: { text: 'Working pattern' }, value: { text: @ect.working_pattern&.humanize } },
-    ].compact_blank
-  )
-%>
-
-<%= govuk_summary_card(title: 'Current training details') do |card|
-  card.with_summary_list(
-    rows: [
-      { key: { text: 'Appropriate body (reported by AB)' }, value: { text: @ect.teacher.induction_periods.last&.appropriate_body&.name } },
-      { key: { text: 'Programme type (reported by AB)' },   value: { text: induction_programme_choice_name(@ect.teacher.induction_periods.last&.induction_programme) } },
-      { key: { text: 'Lead provider (reported by LP)' },    value: { text: @ect.training_periods.last&.provider_partnership&.lead_provider&.name } },
+      { key: { text: 'Working pattern' },   value: { text: @ect.working_pattern&.humanize } },
     ]
   )
-end %>
-
-<%= govuk_summary_card(title: 'School requested details') do |card|
-  card.with_summary_list(
-    rows: [
-      { key: { text: 'Appropriate body' }, value: { text: @ect.appropriate_body&.name } },
-      { key: { text: 'Programme type' },   value: { text: ect_programme_type(@ect) } },
-      { key: { text: 'Lead provider' },    value: { text: @ect.lead_provider&.name } },
-    ]
-  )
-end %>
+%>
+<h2 class='govuk-heading-m'>ECTE training details</h2>
+<%=
+  govuk_summary_card(title: 'Reported to us by your school') do |card|
+    card.with_summary_list(
+      rows: [
+        { key: { text: 'Appropriate body' }, value: { text: @ect.appropriate_body&.name } },
```

Comment:
I missed the `appropriate_body_id` column being added to the `ECTAtSchoolPeriod` model, but for clarity I think this should be more descriptive. @claire-hughez said @ltello suggested `school_recommended_appropriate_body_id`, which I think would fit the bill.

Reply:
To ensure clarity at the db-level, is that because there will also be a corresponding `official_appropriate_body_id` (or some other prefix) on `ECTAtSchoolPeriod`?

And is this the pattern we must follow for `lead_provider_id` too?

There is also a school requested/recommended `programme_type`.

Do we need that in this PR shall we do that separately?